src/binary-reader-interp.cc

```diff
@@ -184,7 +184,7 @@ class BinaryReaderInterp : public BinaryReaderNop {
   wabt::Result OnLoopExpr(Index num_types, Type* sig_types) override;
   wabt::Result OnNopExpr() override;
   wabt::Result OnReturnExpr() override;
-  wabt::Result OnSelectExpr() override;
+  wabt::Result OnSelectExpr(Opcode opcode) override;
```

Comment:
`bitselect` is different enough from `select` to have its own `ExprType`, I think. It actually seems to be a straightforward tertiary operator. If there are other SIMD instructions that have 3 operands, I'd suggest `Tertiary`, otherwise I think just calling it `BitSelect` is fine.

Reply:
Done.